db/migrate/20181001131632_add_conversion_host_id_to_miq_request_tasks.rb

```diff
@@ -0,0 +1,48 @@
+class AddConversionHostIdToMiqRequestTasks < ActiveRecord::Migration[5.0]
+  class MiqRequestTask < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    serialize :options, Hash
+    belongs_to :conversion_host, :class_name => "AddConversionHostIdToMiqRequestTasks::ConversionHost"
+  end
+
+  class ServiceTemplateTransformationPlanTask < MiqRequestTask
+  end
+
+  class ConversionHost < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    has_many :service_template_transformation_plan_tasks, :class_name => "AddConversionHostIdToMiqRequestTasks::MiqRequestTask"
+  end
+
+  class Host < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    has_many :tags, :class_name => "AddConversionHostIdToMiqRequestTasks::Tag"
+  end
+
+  def up
+    add_column :miq_request_tasks, :conversion_host_id, :bigint
+
+    ServiceTemplateTransformationPlanTask.all.reject { |task| task.options[:transformation_host_id].nil? }.each do |task|
+      host = Host.find_by(:id => task.options[:transformation_host_id])
```

Comment:
I think this can use `find` rather than `find_by`. Also this entire block would be way easier to read if `task.options[:transformation_host_id]` was stored in a local variable (ideally before the guard clause).